IronFoundry.Container/ContainerService.cs

```diff
@@ -165,5 +165,10 @@ public IReadOnlyList<IContainer> GetContainers()
         {
             return containers.ToArray();
         }
+
+        public IReadOnlyList<string> GetContainerHandles()
+        {
+            return GetContainers().Select(x => x.Handle).ToList();
```

Comment:
It would be better to just reference `containers` directly here, instead of calling `GetContainers()`.

`GetContainers()` uses `.ToArray()` to create a read-only copy of `containers`.  So we end up with an extra unnecessary allocation in this call.